### ISSUE TITLE
Email addresses are incorrectly converted to links

### ISSUE THREAD

**Issue by vishl** [tombonner/jurlp#5]
When using jrulp to convert links to DOM elements (via $(selector).jurlp) the domain of email addresses is converted to a link.  

For example

```
<div>test@example.com</div>
```

is converted to

```
<div>test@<a href="http://example.com">example.com</a></div>
```

Could I request that it either ignores email addresses, or converts them to mailto: links?

**Comment by tombonner** [comment]
Absolutely. I think that email addresses should be converted to mailto URIs, so I'll implement that logic in the next version.

Thanks for bringing this to my attention.
